src/py/flwr/server/superlink/state/state_test.py

```diff
@@ -707,6 +707,42 @@ def test_store_task_res_task_ins_expired(self) -> None:
         # Assert
         assert result is None
 
+    def test_store_task_res_limit_ttl(self) -> None:
```

Comment:
Should we use `parameterized` to test multiple pairs of `created_at` and `ttl`? This function only test the positive cases, right? I think we should also test negative cases.

Reply:
Yes, good idea

Reply:
I added more test cases, but after a lot of effort, I realized that using `parameterized` tests and `time.time()` makes things flaky. That is why I avoided using it.